src/prophet/model.rs

```diff
@@ -75,6 +78,26 @@ pub enum Language {
     Unknown,
 }
 
+/// Default for language is defined as an unknown language
+impl Default for Language {
+    fn default() -> Self {
+        Language::Unknown
+    }
+}
+
+impl Language {
+    /// TODO figure out how to handle correctly
+    pub fn get_index(lang: &Language) -> usize {
+        match lang {
+            Language::Java => 1,
+            Language::Cpp => 2,
+            Language::Python => 3,
+            Language::Go => 4,
+            Language::Unknown => 0,
+        }
+    }
+}
```

Comment:
This [crate](https://crates.io/crates/enum-ordinalize) exists, if we want to use it.

Reply:
Works for me. I just don't like manually implementing indices like this.